zilencer/auth.py

```diff
@@ -37,6 +39,30 @@
 
 ParamT = ParamSpec("ParamT")
 
+REMOTE_SERVER_TAKEOVER_TOKEN_SALT = "remote_server_takeover"
```

Comment:
@alexmv I don't understand why this would need to be a secret? This isn't all that's used for generating/verifying the signatures - there's an actual `SECRET_KEY` in Django behind the scenes. The salt is just for namespacing essentially

https://docs.djangoproject.com/en/5.1/topics/signing/#using-the-salt-argument
> If you do not wish for every occurrence of a particular string to have the same signature hash, you can use the optional salt argument to the Signer class. Using a salt will seed the signing hash function with both the salt and your [SECRET_KEY](https://docs.djangoproject.com/en/5.1/ref/settings/#std-setting-SECRET_KEY):
> ,,,
> Unlike your [SECRET_KEY](https://docs.djangoproject.com/en/5.1/ref/settings/#std-setting-SECRET_KEY), your salt argument does not need to stay secret.

Reply:
Ahhh, you're 100% right -- I forgot that this implicitly used `SECRET_KEY`.  Sorry!